interface/forms/soap/templates/general_new.html

```diff
@@ -42,6 +72,20 @@ <h2>{xlt t='SOAP'}</h2>
                             </div>
                         </div>
                     </fieldset>
+                    {if empty($BILLING)}
```

Comment:
maybe add a global in the billing category for add billing from soap?

Reply:
No this is not a global check. This is when the provider needs to edit the soap note if billing is submitted already. Don't show the billing fields again. The billing fields will only work the first time the document is opened. I need to configure this so that if they did not do the billing the first time closing the document. It can be done on the edit. The default action is completed the first time the document is opened. On edit, the default action is not used according to my testing. That scenario is not covered by this modification.

Reply:
ok, thanks, would it be ok to add a global so that the default soap doesn't have billing?

Reply:
I asked @bradymiller  about that and he said not to do that because the feature would not be used if it was hidden. It does nothing if the fields are left blank.